gh-functions.sh

```diff
@@ -65,6 +65,14 @@ function ghp() {
                     edit .
                   )
                   ;;
+              -c|--command)
+                  shift
+                  (
+                    cd $PROJ_DIR
+                    ${@}
+                  )
+                  shift $#
+                  ;;                  
```

Comment:
this allow to run any command inside a repo folder, for example: `repo akka/akka/main -c ls`